drivers/tmp006/include/tmp006.h

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (C) 2014 PHYTEC Messtechnik GmbH
+ *               2017 HAW Hamburg
```

Comment:
I like the idea of having most driver code within its subfolder without to much external/distributed dependencies - as `drivers/Makefile.include` already knows about the drivers include dir.